lib/castle/commands/review.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Castle
+  module Commands
+    class Review
+      def initialize(_context)
+      end
```

Comment:
why not just remove the `_context` in init if it's not used?

Reply:
All `Commands` have the same interface, so even if it is not used, it should be there.